.github/workflows/reusable-test.yml

```diff
@@ -294,3 +295,57 @@ jobs:
     - name: Mark run as failed if crash dumps are found
       if: steps.check_dumps.outputs.files_exists == 'true'
       run: exit 1
+
+  create_or_update_issue:
+    needs: run_test
+    if: ${{ failure() }}
```

Comment:
will this file issues due to codecov or is that exempted?

Reply:
This should only file issues when the [reusable-test workflow](https://github.com/microsoft/ebpf-for-windows/blob/main/.github/workflows/reusable-test.yml) fails. Code coverage shows up as a status check, not a workflow failure.